lib/config/config.go

```diff
@@ -603,3 +603,20 @@ func ensureZeroForNodefault(empty interface{}, target interface{}) {
 		return len(v) > 0
 	})
 }
+
+// IgnoresFromFile return Ignores with Lines initiated with the lines of the file at give path.
+func IgnoresFromFile(path string) (Ignores, error) {
+	content, err := ioutil.ReadFile(path)
+	if err != nil {
+		return Ignores{}, err
+	}
+	return Ignores{
+		Lines: strings.Split(string(content), "\n"),
```

Comment:
We probably need to handle Windows line endings here, possibly by just strings.TrimSpace()ing every line

Reply:
Yep. Used `bufio.Scanner` instead to let it handle such issues.